components/engine_rocks/src/misc.rs

```diff
@@ -420,6 +424,21 @@ impl MiscExt for RocksEngine {
             .as_inner()
             .get_approximate_active_memtable_stats_cf(handle))
     }
+
+    fn get_accumulated_flush_count(cf: Option<&str>) -> Result<u64> {
```

Comment:
```suggestion
    fn get_accumulated_flush_count_cf(cf: &str) -> Result<u64> {
         let n = STORE_ENGINE_EVENT_COUNTER_VEC
                .with_label_values(&["kv", cf, "flush"])
                .get();
            Ok(n)
    }
    
    fn get_accumulated_flush_count() -> Result<u64> {
            let mut n = 0;
            for cf in engine_traits::ALL_CFS {
                n += Self::get_accumulated_flush_count_cf(cf)?;
            }
            Ok(n)
    }
```
It's more clear and also follow the existing pattern